ckb-contracts/types/src/config.rs

```diff
@@ -14,10 +14,9 @@ cfg_if::cfg_if! {
         pub const SUDT_HASH_TYPE: u8 = 1;
     } else if #[cfg(feature = "devnet")] {
         pub const SUDT_CODE_HASH: [u8; 32] = [
-            225, 227, 84, 214, 214, 67, 173, 66, 114, 77, 64, 150, 126, 51, 73, 132, 83, 78, 3, 103,
-            64, 92, 90, 228, 42, 157, 125, 99, 215, 125, 244, 25,
+            19, 19, 160, 234, 165, 113, 169, 22, 142, 68, 206, 186, 26, 13, 10, 50, 136, 64, 217, 222, 67, 170, 178, 56, 138, 247, 200, 96, 181, 124, 154, 12,
```

Comment:
how Is it fixed in devnet?

Reply:
If scripts to be deployed fixed, it will be fixed. Otherwise, its value will change.